packages/FormElement/CHANGELOG.md

```diff
@@ -13,3 +13,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 - FormElement no longer adds its extended props to the children components provided. This must now be done by the consumer [@tristanjasper](https://github.com/tristanjasper).
+
+## [0.4.0] - 2020-01-27
```

Comment:
These aren't breaking changes though, so we won't bump up to 0.4.  When the version is less than 1.0, the middle number indicates a breaking change (this is a special case for <1).  So we should only bump to 0.3.2.